django_nitro_mailer/tests/test_tasks.py

```diff
@@ -189,43 +191,43 @@ def test_send_mass_email_success(mock_send_mass_mail: MagicMock, mock_get_connec
 
 @pytest.mark.django_db
 @patch("smtplib.SMTP")
+@override_settings(
+    EMAIL_BACKEND="django_nitro_mailer.backends.SyncBackend",
+    NITRO_MAILER_EMAIL_BACKEND="django.core.mail.backends.smtp.EmailBackend",
+)
```

Comment:
Jeśli to działa, to jestem pod wrażeniem, ale tutaj moja wina, bo wysłałem Ci zły link :sweat: Chodziło mi oczywiście o funkcję od pytest: https://pytest-django.readthedocs.io/en/latest/configuring_django.html#overriding-individual-settings